pkg/build/builder/daemonless.go

```diff
@@ -68,28 +62,17 @@ func pullDaemonlessImage(sc types.SystemContext, store storage.Store, imageName
 		}
 	}
 
-	if registry := sysregistriesv2.FindRegistry(imageName, registries); registry != nil {
-		if registry.Insecure {
-			glog.V(2).Infof("Registry %q is marked as insecure in the registries configuration.", registry.URL)
-			systemContext.DockerInsecureSkipTLSVerify = true
-			systemContext.OCIInsecureSkipTLSVerify = true
-		} else {
-			glog.V(2).Infof("Registry %q is marked as secure in the registries configuration.", registry.URL)
-		}
-	} else {
-		glog.V(2).Infof("Registry for %q is not present in the registries configuration, assuming it is secure.", imageName)
-	}
-
```

Comment:
I assume the designation of secure/insecure registries will be taken care of by copying in `/etc/containers/registries.conf`?

See https://github.com/openshift/origin/pull/21653

Reply:
@nalind did something get fixed in buildah that we don't need this any more?

Reply:
Yes, https://github.com/containers/image/pull/468 was merged.

Reply:
@adambkaplan Yup, the way the builder expects the configuration to be provided to it hasn't changed, we just don't need to do this bit of work ourselves any more.